docker-compose.yml

```diff
@@ -27,10 +27,10 @@ services:
     - DESEC_NS_SIGNALING_DOMAIN_SOA_RNAME
     - DESEC_NS_COOKIES_SECRET
     networks:
-      middle:
+      fmiddle:
         ipv4_address: 10.16.2.2
-      rear:
-      rearreplication:
+      frear:
+      frearreplication:
```

Comment:
While I don't feel very strongly, I think that "telling names" would be better for readability/maintainability (such as `ns_replication`).

Reply:
done, I avoided the `_` though.